client/extensions/woocommerce/app/orders/index.js

```diff
@@ -1,23 +1,44 @@
 /**
  * External dependencies
  */
-import { localize } from 'i18n-calypso';
 import React from 'react';
+import config from 'config';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import ActionHeader from 'woocommerce/components/action-header';
+import Button from 'components/button';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
 import Main from 'components/main';
 import OrdersList from './orders-list';
 
-function Orders( { className, params, translate } ) {
+function Orders( { className, params, site, translate } ) {
+	let addButton = null;
+
+	if ( config.isEnabled( 'woocommerce/extension-orders-create' ) ) {
+		addButton = (
+			<Button primary href={ getLink( '/store/order/:site/', site ) }>
+				{ translate( 'New Order' ) }
+			</Button>
+		);
+	}
+
 	return (
 		<Main className={ className }>
-			<ActionHeader breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
+			<ActionHeader breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) }>
```

Comment:
:disappointed: 100 existing translations will be lost with this change.
If the string must change, the following option has already been translated **96** times:
`translate( 'Order:' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).